corehq/apps/export/management/commands/migrate_exports.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand
+
+from corehq.apps.reports.dbaccessors import (
+    stale_get_exports_json,
+    stale_get_export_count,
+)
+from corehq.apps.export.utils import convert_saved_export_to_export_instance
+from corehq.apps.domain.models import Domain
+from corehq.util.log import with_progress_bar
+
+
+class Command(BaseCommand):
+    help = "Migrates old exports to new ones"
+
+    def handle(self, *args, **options):
+
+        for doc in Domain.get_all(include_docs=False):
+            domain = doc['key']
+            export_count = stale_get_export_count(domain)
+            if export_count:
+                for old_export in with_progress_bar(
+                        stale_get_exports_json(domain),
+                        length=export_count,
+                        prefix=domain):
+                    try:
+                        convert_saved_export_to_export_instance(domain, old_export)
```

Comment:
Could use gevent to spawn multiple of these tasks, but don't think it's worth it. I suspect there aren't that many exports and this is only going to be run once